packages/material-ui-system/src/index.d.ts

```diff
@@ -219,5 +219,8 @@ export type TypographyProps = PropsFor<typeof typography>;
 
 export const visuallyHidden: React.CSSProperties;
 
+export const getThemeValue: (prop: string, value: any, theme: object) => any;
+export const styleFunctionSx: (props: object) => object;
```

Comment:
Should we name it just `sx`?

Reply:
I like this verbose name, it will make it easier to search in a codebase.